liquibase-core/src/main/java/liquibase/diff/output/changelog/core/ChangedSequenceChangeGenerator.java

```diff
@@ -37,40 +38,70 @@ public int getPriority(Class<? extends DatabaseObject> objectType, Database data
     public Change[] fixChanged(DatabaseObject changedObject, ObjectDifferences differences, DiffOutputControl control, Database referenceDatabase, Database comparisonDatabase, ChangeGeneratorChain chain) {
         Sequence sequence = (Sequence) changedObject;
 
+        AlterSequenceChange accumulatedChange = createAlterSequenceChange(sequence, control);
```

Comment:
Is there a reason we have to treat postgresql differently with this and have the single Change? While it is a bit nicer to have just one statement executed, I'm not sure it's worth the difference in implementation

Reply:
I failed to understand what is the reason for split and added join is to solve validation problem.
a lot of sequence properties are codependent -- startAt must be bigger than minValue, both minValue and maxValue must be in range of dataType, etc.
and alter will fail if we try to execute our 'atomic' alter in wrong order.
I didn't attempt to reorder, because its conditional -- one may upgrade dataType and maxValue, so must alter dataType first, but one may move backward (if original setup was too greedy).
one can do math and determine order on the fly, but I don't understand whats are advantages for this way of doing things.

Reply:
The reason we do each change as a separate alter is because some database types don't allow you to change all the different sequence settings at one time. Rather than having two codepaths it seemed safer to have one code path that would work consistently across all the platforms. 

But, if postgresql complains if you are setting invalid values along the way, then we may need to have that split path. It's probably best to have that "do it in one statement" code not just around postgresql directly but around a "canBundleChanges" variable that is currently only set to true for postgresql but can be expanded to other databases as we get around to testing them.